WordPress/Classes/ViewRelated/NUX/SigninAccountAndBlogViewController.swift

```diff
@@ -0,0 +1,656 @@
+import UIKit
+import WordPressComAnalytics
+import WordPressShared
+
+
+/// Create a new WordPress.com account and blog.
+///
+@objc class SigninAccountAndBlogViewController : NUXAbstractViewController, SigninKeyboardResponder
```

Comment:
I know you're using the `Signin` prefix for all the new classes, but this class name is really confusing. Can we just do `SignupViewController`?